src/workerd/jsg/setup.c++

```diff
@@ -562,6 +562,22 @@ void IsolateBase::jitCodeEvent(const v8::JitCodeEvent* event) noexcept {
   }
 }
 
+void* getJsCageBase() {
+  if (!v8Initialized) return nullptr;
+  v8::Isolate* isolate = v8::Isolate::TryGetCurrent();
+  if (isolate == nullptr) return nullptr;
+  // Returns null if we are not using pointer cages.
+  return isolate->GetData(4);
```

Comment:
Where does the magic number 4 come from? Can it be hoisted out to a constexpr variable?